src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,89 @@
+import usersFromServer from '../../api/users';
+
+type Props = {
+  onSubmit: (event: React.FormEvent) => void,
+  title: string;
+  setTitle: (value: string) => void,
+  userId: number;
+  setUserId: (value: number) => void,
+  titleError: string;
+  setTitleError: (value: string) => void,
+  userIdError: string,
+  setUserIdError: (value: string) => void,
```

Comment:
These error states don't need to be located in App. You can store them directly in TodoForm

Reply:
Because, they are not used by anything else but TodoForm